packages/glibc.rb

```diff
@@ -590,6 +590,14 @@ def self.postinstall
     Dir.chdir CREW_LIB_PREFIX do
       puts "System glibc version is #{LIBC_VERSION}.".lightblue
       puts 'Creating symlinks to system glibc version to prevent breakage.'.lightblue
+      case ARCH
+      when 'aarch64', 'armv7l'
+        FileUtils.ln_sf "/lib/ld-#{LIBC_VERSION}.so", 'ld-linux-armhf.so.3'
+      when 'i686'
+        FileUtils.ln_sf "/lib/ld-#{LIBC_VERSION}.so", 'ld-linux-i686.so.2'
+      when 'x86_64'
+        FileUtils.ln_sf "/lib64/ld-#{LIBC_VERSION}.so", 'ld-linux-x86-64.so.2'
+      end
```

Comment:
Why are we doing this in postinstall instead of install so the symlinks can be deleted after the package is removed?

Reply:
The postinstall handles situations where we don't have a binary package for a glibc build yet, same as the rest of the symlinks created in the postinstall.

Reply:
I guess the next question is why don't we have binary packages yet?  I'm just thinking about potential dangling symlinks during upgrades and the like.

Reply:
Just added binaries.